### ISSUE TITLE
fix yop project for latest VScode release

### ISSUE THREAD

**Issue by jurgenvinju** [usethesource/rascal-language-servers#434]
**Describe the bug**

<img width="1029" alt="image" src="https://github.com/user-attachments/assets/48198f92-02fb-494c-a989-da23bd154716">

This does not break the code, but since we have ultimate beginners looking at these files, it could be confusing to read the new (otherwise very helpful) message.

**Comment by DavyLandman** [comment]
I don't think this is a VS Code bug right? Looks more like a specific example project that needs to be updated?

**Comment by jurgenvinju** [comment]
I posted it here to signal that we may have some (important) downstream projects that are now broken with the latest VScode extension without us realizing it. This one has hundreds of users, for example. 

Is a missing pom.xml file really breaking the extension? I think we might need a _softer message_ for a missing pom file, as compared to a missing RASCAL.MF file.

**Comment by DavyLandman** [comment]
It's already a warning, but yes, if you are missing a pom.xml, some of your dependencies might not be picked up. (like say you did edit your rascal.mf to have a dependency).

Can we make better diagnostics? Yes. I would also ideally like us to have a quickfix for the problems we know how to fix..

**Comment by DavyLandman** [comment]
@jurgenvinju I think the latest fixes to the validators fixed this, can you make sure?

**Comment by DavyLandman** [comment]
@jurgenvinju I'm closing the issue, unless it still doesn't work.
